src/Templates/SingleProductTemplateCompatibility.php

```diff
@@ -396,10 +392,11 @@ private static function create_wrap_block_group( $blocks ) {
 	 * woocommerce/product-gallery-image, woocommerce/product-details, woocommerce/add-to-cart-form]
 	 *
 	 * @param array $parsed_blocks Array of parsed block objects.
-	 * @param array $single_product_template_blocks Array of single product template blocks.
 	 * @return bool True if the template has a single product template block, false otherwise.
 	 */
-	private static function has_single_product_template_blocks( $parsed_blocks, $single_product_template_blocks ) {
+	private static function has_single_product_template_blocks( $parsed_blocks ) {
+		$single_product_template_blocks = array( 'woocommerce/product-image-gallery', 'woocommerce/product-details', 'woocommerce/add-to-cart-form', 'woocommerce/product-meta', 'woocommerce/product-price', 'woocommerce/breadcrumbs' );
```

Comment:
What do you think about adding an attribute or context that indicates a given block is part of the single product template instead of hardcoding names here? 

The reason I'm suggesting this change is because this list can quickly become obsolete as more blocks are added and/or their usage is enabled within the single product template.

Reply:
Done! https://github.com/woocommerce/woocommerce-blocks/issues/10008